content/css/concepts/grids/grids.md

```diff
@@ -13,7 +13,7 @@ CatalogContent:
   - 'paths/front-end-engineer-career-path'
 ---
 
-CSS Grid is a two-dimensional grid-based layout system, with rows, columns and gaps. This layout system makes it easier to design and fine-tune the layout of web pages without having to use positioning and floats.
+CSS Grid is a two-dimensional grid-based layout system, with rows, columns and gaps. This layout system makes it easier to design and fine-tune the layout of web pages without having to use positioning and floats. CSS grids are better becuase it has a lot more options to customisation compared to HTML tables.
```

Comment:
```suggestion
CSS Grid is a two-dimensional grid-based layout system with rows, columns, and gaps. This layout system makes it easier to design and fine-tune the layout of web pages without having to use positioning and floats.
```

Agree with your point that when using CSS grid, there are lots of options for customization when deciding on the layout of a web page. However, I'm not sure if HTML tables is a great comparison for grid.
CSS Float is more comparable, but even then everyone has their preference for different reasons. I'd definitely look into this a bit more and the general discourse that exists on which parts of each technique are favored.